blindsign/blindrsa/rsa.go

```diff
@@ -0,0 +1,149 @@
+package blindrsa
```

Comment:
does this file came from go stdlib? if so, then add the license header.

Reply:
Yeah, this came from the standard library.

Reply:
I added the license to each of the files.